src/reflect/scala/reflect/api/Names.scala

```diff
@@ -30,16 +30,16 @@ import scala.language.implicitConversions
  */
 trait Names {
   /** An implicit conversion from String to TermName.
-   *  Enables an alternative notation `"map": TermName` as opposed to `TermName("map")`.
-   *  @group Names
-   */
+  * Enables an alternative notation `"map": TermName` as opposed to `TermName("map")`.
```

Comment:
This is not a deprecation change, and it makes the formatting inconsistent with the rest of the file